### ISSUE TITLE
In multi-card applications, Device_utilization_desc_of_container metrics only has data on one card.

### ISSUE THREAD

**Issue by CoderTH** [Project-HAMi/HAMi#445]
_The template below is mostly useful for bug reports and support questions. Feel free to remove anything which doesn't apply to you and add more information where it makes sense._
---

### 1. Issue or feature description

In multi-card applications, Device_utilization_desc_of_container metrics only has data on one card.

![ZMSGuEpzJ8](https://github.com/user-attachments/assets/c6739fa5-0d0f-4230-aef4-ffaf61c32e3b)

![img_v3_02dt_8c2f42eb-6307-49f8-bdc4-f1825301cffg](https://github.com/user-attachments/assets/6b27a83a-f4b1-45f5-9bc0-8f847ee264c5)

### 3. Information to [attach](https://help.github.com/articles/file-attachments-on-issues-and-pull-requests/) (optional if deemed irrelevant)

Common error checking:
- [ ] The output of `nvidia-smi -a` on your host
- [ ] Your docker or containerd configuration file (e.g: `/etc/docker/daemon.json`)
- [ ] The vgpu-device-plugin container logs
- [ ] The vgpu-scheduler container logs
- [ ] The kubelet logs on the node (e.g: `sudo journalctl -r -u kubelet`)

Additional information that might help better understand your environment and reproduce the bug:
- [ ] Docker version from `docker version`
- [ ] Docker command, image and tag used
- [ ] Kernel version from `uname -a`
- [ ] Any relevant kernel output lines from `dmesg`

**Comment by archlitchi** [comment]
you're right, current hami-core only count gpu utilization on GPU0, that's an issue needs to be solved
